install_arrow.sh

```diff
@@ -0,0 +1,21 @@
+sudo apt update
```

Comment:
I'm not sure what this is for

Reply:
holdover from an experiment in configuring linux env, removed